default-config.json

```diff
@@ -60,6 +60,16 @@
         "    2. Provided the name of a command (`%helpTopic help`), it will provide details regarding the specific usage of that command."
       ]
     },
+    "nowplaying": {
+      "internalKey": "nowplaying",
+      "subtitle": "The nowplaying command displays the song that is currently playing on Cadence.",
+      "lines": [
+        "CadenceBot will query the Cadence server for the song that is current playing and then reply to the command with the song name and artist."
```

Comment:
I hate the fact that I cannot think of better text than this.

Except perhaps to make it even less specific? I doubt we'd ever actually change the format of now playing results...

Reply:
Ok, so how about "CadenceBot will query the Cadence server for the current playing song and then reply to the command with the song."

Reply:
I prefer what you have now if only because I dislike repeating the word "song" so much.

"CadenceBot will query the Cadence server for the currently playing song and display its information?"

Of course, suddenly the subtitle and body look very similar....

Reply:
I like how it looks. Succinct, but descriptive.

Reply:
Updated based on feedback above and the Cadence testing server.